shared/Microsoft.Extensions.ActivatorUtilities.Sources/ActivatorUtilities.cs

```diff
@@ -353,5 +416,15 @@ public object CreateInstance(IServiceProvider provider)
                 }
             }
         }
+
+        private static void ThrowMultipleCtorsMarkedWithAttributeException()
+        {
+            throw new InvalidOperationException("Multiple constructors were marked with " + nameof(ActivatorUtilitiesConstructorAttribute));
+        }
+
+        private static void ThrowMarkedCtorDoesNotTakeAllProvidedArguments()
+        {
+            throw new InvalidOperationException("Constructor marked with " + nameof(ActivatorUtilitiesConstructorAttribute) + " does not accept all given argument types.");
```

Comment:
no love for interpolated string?

Reply:
Interpolations are not converted into consts (https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQBMQGoA+ABADAAmwEYBuAWAChsBmQgJnwGF8BvS/Dw27AFnwFkAFAEpW7ThOIBOQQBIARCwB2EMHAD2AM0H9hAX3nDyFCXsp6gA) like concats are, but I guess it's less important here.